tests/src/com/unciv/testing/BasicTests.kt

```diff
@@ -190,14 +190,70 @@ class BasicTests {
         Assert.assertTrue("This test succeeds only if all uniques are presented in UniqueType.values()", allOK)
     }
 
+    @Test
+    fun allPolicyRelatedUniquesHaveTheirUniqueTypes() {
+        val policies = ruleset.policies.values
+        val policyBranches = ruleset.policyBranches.values
+        var allOK = true
+        for (policy in policies) {
+            for (unique in policy.uniques) {
+                if (!UniqueType.values().any { it.placeholderText == unique.getPlaceholderText() }) {
+                    println("${policy.name}: $unique")
+                    allOK = false
+                }
+            }
+        }
+
+        for (policyBranch in policyBranches) {
```

Comment:
I think that ruleset.policies already contains all policy branches

Reply:
Deleted the duplicate check.